pandas/conftest.py

```diff
@@ -1903,6 +1903,7 @@ def using_copy_on_write() -> bool:
     """
     Fixture to check if Copy-on-Write is enabled.
     """
+    pd.options.mode.copy_on_write = True
```

Comment:
I suppose this is a leftover from local testing?

Reply:
Yes, thanks for noticing

Reply:
BTW, for testing, you can set the env variable `PANDAS_COPY_ON_WRITE=1 pytest ...`, which is of course a bit verbose. What I do locally is that I have an alias in my bashrc, so I can just do `COW pytest ...`, and then it is quite easy to switch between `COW pytest ...` or `pytest ...` when I want to run a specific test while developing with both COW enabled or not.

Reply:
Yeah agreed, did this when running tests in PyCharm

Reply:
Ah, I can't speak for PyCharm ;)